src/hooks/Projects.ts

```diff
@@ -141,6 +148,36 @@ export function useProjectsSearch(handle: string | undefined) {
   )
 }
 
+/**
+ * Search sepana projects for query and return only a list of projects
+ * @param text text to search
+ * @param pageSize number of projects to return
+ * @returns list of projects
+ */
+export function useSepanaProjectsSearch(
+  text: string | undefined,
+  pageSize?: number,
+) {
+  return useQuery(
+    ['sepana-query', text, pageSize],
+    () =>
+      axios
+        .get<SepanaSearchResponse<SepanaProjectJson>>(
+          `/api/sepana/projects?text=${text}${
+            pageSize !== undefined ? `&pageSize=${pageSize}` : ''
+          }`,
+        )
+        .then(res =>
+          res.data.hits.hits.map(
```

Comment:
**question** is it possible that any of this data could be undefined? Is it worth optional-chaining here?

Reply:
doesn't seem like it, but down to error on the side of caution here